### DISCUSSION TITLE
Setting a relative error value for pppm calculations

### DISCUSSION THREAD

**Discussion by amruthesht** [glotzerlab/hoomd-blue#1764] [Support] [answered]
Hi All,

I had a generic question on whether hoomd offers the functionality to set a relative error or accuracy value for forces calculated using the [`make_pppm_coulomb_forces`](https://hoomd-blue.readthedocs.io/en/latest/module-md-long_range-pppm.html#hoomd.md.long_range.pppm.make_pppm_coulomb_forces) module for long-range interactions. I couldn't find anything in the documentation regarding this. LAMMPS users typically avail this through the [`kspace_style`](https://docs.lammps.org/kspace_style.html) command

If hoomd right now doesn't offer this option directly, has anyone been able to modify the source code to come up with this functionality?

Thank you!
-
Amru

**Comment by joaander** [comment, marked as answer]
HOOMD-blue does not offer this functionality. I would welcome a pull request from an expert on the intricacies of the PPPM method.
